go.mod

```diff
@@ -1,5 +1,7 @@
 module github.com/livesense-inc/fanlin
 
+go 1.23.5
```

Comment:
I bumped go version to the latest because an error has occurred at `dominikh/staticcheck-action` action in workflow.